### PR TITLE
feat(i18n): Add i18n support

### PR DESCRIPTION
**Pull request by TheLearneer**
# About

Added i18n support in the bot with `en-US` as default language.
After we add configuration support to change language for a server, we can set `ne-NP` as the default langauge.

## NOTE
I have only configured some of the text used throughout the bot, we need to work on others.

### CODE REVIEW

File: src/locales/ne-NP/commands.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "laugh": {
+        "random": "{{user1}} आफैँ एक्लै हास्दै हुनुन्छ! मज्जाको जोक याद आएको हुनु पर्छ।",
+        "laughing": "{{user1}} {{user2}}लाई हेरेर हास्नु भयो।"
+    },
+    "slap": {
+        "self": "आफैले आफुलाई झाप्पड किन हान्नु पर्यो र?",
+        "slapping": "{{user1}}ले {{user2}}लाई झाप्पड हान्नु भयो। ऐया, धेरै दुख्यो होला।"
+    },
+    "ronb": {
+        "viewOnTwitter": "यो पोस्ट Twitter मा हेर्नुहो।",
```
Comment (DevAndromeda) [inline src/locales/ne-NP/commands.json:23]: shouldn't it be `यो पोस्ट Twitter मा हेर्नुहोस्।`?

File: src/locales/ne-NP/default.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "commands": {
+        "failedToExecute": "{command} कमाण्ड चलाउन सकिएन|"
```
Comment (DevAndromeda) [inline src/locales/ne-NP/default.json:3]: I guess `कमान्ड` looks better

File: src/locales/ne-NP/commands.json (hunk 1 above)
Comment (TheLearneer) [inline src/locales/ne-NP/commands.json:23]:
```suggestion
        "viewOnTwitter": "यो पोस्ट Twitter मा हेर्नुहोस्।",
```

File: src/locales/ne-NP/default.json (hunk 2 above)
Comment (TheLearneer) [inline src/locales/ne-NP/default.json:3]:
```suggestion
        "failedToExecute": "{command} कमान्ड चलाउन सकिएन|"
```

**Review by TheLearneer — COMMENTED** [review]
Changes to make

**Review by DevAndromeda — APPROVED** [review]

**Pull request closed without merging**